spec/config/RpcNodeTestConfig.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Prefer a default export instead of a module export here

Reply:
same. for some reason that doesn't work with jest. probably es6 related stuff.